test/e2e/framework/util.go

```diff
@@ -3782,22 +3782,31 @@ type SSHResult struct {
 // is no error performing the SSH, the stdout, stderr, and exit code are
 // returned.
 func SSH(cmd, host, provider string) (SSHResult, error) {
+	var (
+		stdout string
+		stderr string
+		code   int
+		err    error
+	)
 	result := SSHResult{Host: host, Cmd: cmd}
+	if provider == "vsphere" {
```

Comment:
This concept does scale well with multiple providers. Why is vsphere unique here?